Gemfile.lock

```diff
@@ -113,6 +113,7 @@ GEM
     parallel (1.12.0)
     parser (2.4.0.0)
       ast (~> 2.2)
+    orm_adapter (0.5.0)
```

Comment:
There were conflicts with the master once I approved the Pull Request for Devise. I "resolved" the conflicts by keeping both what was in the master Gemfile and what was in this branch Gemfile. As an afterthought I'm not sure this was the right thing to do, maybe there are incompatibilities.